File: HomeWork/Home10.1/src/main/Team.java
public class Team {
    String name;
    int goalsScored = 0;
    int goalsConceded = 0;
    int yellowCards = 0;
    int redCards = 0;
    double topHeight = 0.0;
    double points = 0.0;

    public Team(String name) {
        this.name = name;
    }

    public void updateStats(int scored, int conceded, int yellow, int red, double height) {
        goalsScored += scored;
        goalsConceded += conceded;
        yellowCards += yellow;
        redCards += red;
        topHeight += height;
    }
}

File: Class/practice10.1/Team.cpp
#include "Team.h"

Team::Team(const std::string &name) : name(name) {}

void Team::updateStats(int scored, int conceded, int yellow, int red, double height) {
    goals_scored += scored;
    goals_conceded += conceded;
    yellow_cards += yellow;
    red_cards += red;
    top_height += height;
}
